fastchat/llm_judge/common.py

```diff
@@ -39,6 +39,28 @@
         "gpt-4",
         "claude-v1",
     ],
+    # "mt_bench": [
```

Comment:
use glob or other method to automatically detect this list?

Reply:
fixed!